INSTALL_DOCKER.rst

```diff
@@ -32,15 +32,15 @@ If your user does not already have ``sudo`` rights, you should add your user to
 
   ::
 
-    usermod -G wheel -a <USERNAME>
+    $ usermod -G wheel -a <USERNAME>
```

Comment:
not here I think - the instructions note you should be logged in as root I believe?